backend/app.js

```diff
@@ -1,9 +1,40 @@
 const express = require("express");
-// Creating the node app
+const mongoose = require('mongoose');
+const cors = require('cors');
+
+require('dotenv').config();
+
 const app = express();
-// Port Environment variable
 const port = process.env.PORT || 5000;
 
+app.use(cors());
+app.use(express.json());
+
+MONGODB_URL = 'mongodb+srv://shir:shir@cluster0.oxayx.mongodb.net/myFirstDatabase?retryWrites=true&w=majority';
```

Comment:
you should leave the `connect` function in a separate file, as it was, and you  should use the generic way to connect the DB  `MONGODB_URL = 'mongodb://localhost:27017/mydb'` like it was before